### ISSUE TITLE
MIT License

### ISSUE THREAD

**Issue by jfizz** [JoshData/jot#14]
Hi! I am looking to add collaborative features to an online, image editor and would love to use jot. However, I won't be able to use it while it is licensed under GPL v3 due to the "Disclose source" condition. Would it be possible to move to a MIT license?

Regardless, great work on the library and thanks for your time.

**Comment by JoshData** [comment]
Hi there. Thanks for your interest!

I'd be glad to grant you a license at no charge for your image editor --- I want people to be able to use JOT.  It's just that at this time I'm not ready to open source it to keep my options open. But I am totally happy to grant licenses on a case by case basis like this. If you'd like to do that, drop me an email at jt@occams.info?

This goes for anyone else reading this issue in the future too. Please go ahead an email me if the current license won't work for you. :)

**Comment by bionicles** [comment]
+1 for MIT

**Comment by emcmanus** [comment]
Hi Josh, thanks for making JOT! It looks useful, and judging by the project's star count many others feel the same way. I was hoping you could clarify the thinking behind GPL. Is the intent for JOT to be used exclusively in other GPL projects?

My limited understanding of GPL is it's unclear whether distribution to a browser constitutes "distribution" under the terms of the license, so it's a nonstarter for most commercial projects. Which may be desirable! But I figured I'd ask.

**Comment by JoshData** [comment]
Thanks. Star count aside, I don't know if anyone has actually demonstrated that the library (correctly) solves a real problem. If it does (and in particular if it's of commercial value), I would be pleased to discuss (over email) granting an alternative license for your organization.

My intention/understanding with GPL is that it would only allow use within applications in which all other components and libraries of the application are GPL as well.

**Comment by emcmanus** [comment]
> My intention/understanding with GPL is that it would only allow use within applications in which all other components and libraries of the application are GPL as well.

Correct. You would also need permission from your contributors to issue alternative licenses. Anyhow, it sounds like you know what you're doing, and GPL wasn't just selected at random, I appreciate you getting back to me!

**Comment by JoshData** [comment]
> You would also need permission from your contributors to issue alternative licenses.

That wouldn't be a problem.
